centos.sh

```diff
@@ -155,6 +155,8 @@ build_deps(){
                         INSTALLED+=("bzip2-devel")
                     elif [ "$FOUND_VALUE" = "libssh2" ]; then
                         INSTALLED+=("libssh2-devel")
+                    elif [ "$FOUND_VALUE" = "libxml2" ]; then
```

Comment:
We can also bundle libxml2, but it is a pretty common library, so I left it an external dependency for the time being.

Reply:
libxml2 is bundled in the new version.